package.json

```diff
@@ -5,17 +5,23 @@
   "main": "index.js",
   "types": "index.d.ts",
   "exports": {
-    "types": "./index.d.ts",
-    "import": "./index.mjs",
-    "require": "./index.js"
+    ".": {
+      "import": "./index.mjs",
+      "require": "./index.js",
+      "types": "./index.d.ts"
+    },
+    "./utilities": {
+      "import": "./utilities/index.mjs"
+    }
```

Comment:
reformatted w/ the same overall meanings to ensure we could do both split packages for esm/cjs & do subpackage exports